static/js/chart/draw.ts

```diff
@@ -213,6 +213,80 @@ function drawSingleBarChart(
     .attr("fill", (d) => color(d.label));
 }
 
+/**
+ * Create a canvas object with a color gradient to be used as a color scale.
+ * @param color  d3.scaleLinear object that encodes the desired color gradient.
+ * @param n Number of color tones to transition between.
+ */
+function ramp(color, n = 256) {
```

Comment:
Ah, I had meant to have this in dev.ts (so leaving draw.ts untouched) so you won't have to worry about exports and such. Is the plan to move these functions to one of the choropleth modules? Please do that in this pr, or create a new file in this folder for choropleth drawing related code.

Also, please give this a more descriptive function name.

Reply:
My plan was to move all of this directly to `choropleth.tsx`.

Reply:
ok, let's do that right away then. we can close this pr (all comments are still valid, and thanks for addressing them).